server/lib/subscriptions/subscriptionMapper.js

```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash')
+const { withFilter } = require('graphql-subscriptions')
+const { evalWithContext } = require('../util/filterEvaluator')()
+const { log } = require('../util/logger')
+
+module.exports = function mapSubscriptions (subsciptionMappings, pubsub) {
```

Comment:
I would add some comments on what this module is doing.

Reply:
Great suggestion